web/genericcom.js

```diff
@@ -56,9 +56,47 @@ class MLManager {
     return null;
   }
 
-  async guess() {
+  isReady(_name) {
+    return false;
+  }
+
+  guess() {}
```

Comment:
`guess(_data) {}` I assume, given that the `guess` method in `firefoxcom` has a `data` argument?